src/merge.ts

```diff
@@ -30,7 +120,30 @@ export const mergeOperations = mergeLists<IHttpOperation[]>(
     ...o1,
     request: {
       ...o1.request,
-      headers: mergeHeaders(o1.request?.headers || [], o2.request?.headers || []),
+      path:
+        o1.request?.path && o2.request?.path
+          ? mergeParams(o1.request?.path, o2.request?.path)
```

Comment:
Shall we merge path parameters? Wouldn't this also change the final path of the operation?

`p1: /a/{param1}/`
`p2: /b/{param2}/`

What would be the merged path?

Reply:
Well.. it may be a bit impossible to create such case, see: https://github.com/stoplightio/http-spec/pull/110/files#diff-108eac4f51e2b99cad5e9affa181be19R118. The paths will be the same anyway before executing merge procedure (procedure :trollface:).

So currently, for postman, this is exactly the same as just writing: `path: o1.request?.path`. I'm ok with leaving it like that or taking all path params from `o1`.